### ISSUE TITLE
Menu: Reset Button

### ISSUE THREAD

**Issue by MSpelman** [MSpelman/Inspire#15]
Once a scan is complete, a new calibration is needed in order to run another scan. This is to reset the system between patients, so the Tech cannot use the calibration from a previous patient while doing a scan for a different patient.

However, there could be cases where a patient has a calibration, but does not go through with the scan. In this case the Tech can always just do another calibration for the next patient. However, it would be nice to have a Reset button to clear out the old calibration to make sure the calibration from the patient who did not go through with their scan was not accidentally used for a subsequent patient.

Also, if they don't like the fact the system resets after a scan, they can always use this button to reset the system between patients instead (it is a one line change to stop the auto-reset).
